quetz/utils.py

```diff
@@ -24,6 +30,74 @@ def add_entry_for_index(files, subdir, fname, data_bytes):
     )
 
 
+def parse_query(query):
+    accepted_filters = ['channel', 'description', 'summary']
+    query = unquote(query.strip())
```

Comment:
do you need it? I would assume that fastapi would already do it for you

Reply:
Yes, we are mimicking of what we would ideally receive from text entered into a search box. We can anyway remove this later if we figure out that FastAPI does this OR perhaps if the content entered into a search box stays the same.